ui/tests/integration/components/auth-form-test.js

```diff
@@ -45,19 +35,44 @@ module('Integration | Component | auth form', function (hooks) {
   hooks.beforeEach(function () {
     this.owner.register('service:router', routerService);
     this.router = this.owner.lookup('service:router');
+    this.auth = this.owner.lookup('service:auth');
+    this.cluster = EmberObject.create({ id: '1' });
+    this.selectedAuth = 'token';
+    this.performAuth = sinon.spy();
     this.onSuccess = sinon.spy();
+    this.renderComponent = async () => {
+      return render(hbs`
+        <AuthForm
+          @wrappedToken={{this.wrappedToken}}
+          @cluster={{this.cluster}}
+          @selectedAuth={{this.selectedAuth}}
+          @performAuth={{this.performAuth}}
+          @delayIsIdle={{this.delayIsIdle}}
+        />`);
+    };
+
+    this.renderParent = async () => {
```

Comment:
Tests that use `this.renderParent()` will be moved to a separate component integration test for the new `auth/page.js` component. For now, I've stubbed the component here to be sure we didn't lose any test coverage/functionality with the refactor.